Dockerfile

```
@@ -0,0 +1,25 @@
+# This file permits OpenRA to be built on Windows without the need to install
+# any local development tools or SDKs. From the repository root, build using:
+#
+# docker build -t openra .
+# 
+# You can then open the container and build interactively using:
+#
+# docker run -it -v .:C:\OpenRA openra
+#
+# You can now run `./make.cmd`: output will be shared to the host.
+#
+# Alternatively, build directly from the host by running:
+# 
+# docker run -it -v .:C:\OpenRA openra ./make.ps1 a
+
+FROM mcr.microsoft.com/windows/servercore:ltsc2022
```

You can use
```suggestion
FROM mcr.microsoft.com/dotnet/sdk:6.0
```
which will then use a slimmer Linux-based image that does not require a Windows Server license and probably also highlights how mad this actually is.

---

The problem with using a Linux image is that so far as I can see, there is no cross-compilation support in the current build system: it will always build Linux executables (and depend upon e.g. SDL) if the host OS is Linux. If it's possible to build Windows releases from Linux already then this is indeed unnecessary and I'd much appreciate a pointer to where it is documented.

---

Take a look at https://github.com/OpenRA/OpenRA/blob/bleed/packaging/windows/buildpackage.sh as a starting point.

---

Thanks a lot! This does indeed point me in the right direction. I'll close this PR for now while I work on a better Docker story using the existing cross-compilation option.